scripts/initiate_ritual.py

```diff
@@ -83,7 +83,7 @@ def cli(
     num_nodes,
     random_seed,
     handpicked,
-    min_version
+    min_version,
```

Comment:
Shouldn't the trailing comma be the default linting?

Reply:
Indeed, it should be - it was missed previously for some reason.

Reply:
Oh I don't know why I interpreted the diff the other way around. Disregard...